content/doc/upgrade-guide/2.73.adoc

```diff
@@ -9,6 +9,10 @@ notitle: true
 Each section covers the upgrade from the previous LTS release, the section on 2.73.1 covers the upgrade from 2.60.3.
 
 
+=== Upgrading to Jenkins LTS 2.73.3
+
+No notable changes requiring upgrade notes.
```

Comment:
"Stop using JDKInstaller, please"

Reply:
If consensus is that this is a terrible idea, should this be an admin monitor?